src/scales/interpolatedColorScale.ts

```diff
@@ -96,8 +96,8 @@ export class InterpolatedColor extends Scale<number, string> {
   /**
    * Generates the converted QuantitativeScale.
    */
-  private _d3InterpolatedScale() {
-    return this._colorScale.range([0, 1]).interpolate(this._interpolateColors());
+  private _d3InterpolatedScale(): supportedScale {
```

Comment:
nit: I'd suggest renaming the local type def to `SupportedScale` in this file

```suggestion
  private _d3InterpolatedScale(): SupportedScale {
```